pages/search.tsx

```diff
@@ -0,0 +1,68 @@
+import { MarketStatus } from "@zeitgeistpm/indexer";
+import { useMarketSearch } from "lib/hooks/queries/useMarketSearch";
+import { NextPage } from "next";
+import Link from "next/link";
+import { useState } from "react";
+import { X } from "react-feather";
+
+const SearchPage: NextPage = () => {
+  const [searchTerm, setSearchTerm] = useState("");
+
+  const { data: markets } = useMarketSearch(searchTerm);
+  return (
+    <div className="mt-4">
+      <div className="flex items-center">
+        <input
+          className="rounded-sm h-8 px-2 w-full focus:outline-none max-w-[500px] border border-sky-200"
+          value={searchTerm}
+          placeholder="Search markets"
+          onChange={(event) => {
+            setSearchTerm(event.target.value);
+          }}
+        />
+        <button
+          className="relative right-6 text-sky-600"
+          onClick={() => {
+            setSearchTerm("");
+          }}
+        >
+          <X size={16} />
+        </button>
+      </div>
+      {markets && (
+        <div className="flex flex-col py-4">
+          <div className="text-sky-600">Results</div>
+
+          {markets.length > 0 ? (
+            markets?.map((market) => (
+              <Link
```

Comment:
Would be nice to have a hover effect here too so you can see which result is being highlighted